mimic/model/flavor_objects.py

```diff
@@ -71,6 +66,214 @@ def detailed_json(self, absolutize_url):
             "links": self.links_json(absolutize_url),
             "name": self.name,
             "ram": self.ram,
+            "vcpus": self.vcpus,
+            "rxtx_factor": self.rxtx,
+            "disk": self.disk,
             "OS-FLV-WITH-EXT-SPECS:extra_specs": self.extra_specs_json()
         })
         return template
+
+
+class RackspaceStandardFlavor(Flavor):
+    """
+    A Rackspace standard flavor object representation
+    """
+    flavors = {"512 MB Standard Instance": {"id": "2", "ram": 512, "vcpus": 1, "rxtx_factor": 80,
+                                            "disk": 20},
+               "1 GB Standard Instance": {"id": "3", "ram": 1024, "vcpus": 1, "rxtx_factor": 120,
+                                          "disk": 40},
+               "2 GB Standard Instance": {"id": "4", "ram": 2048, "vcpus": 2, "rxtx_factor": 240,
+                                          "disk": 80},
+               "4 GB Standard Instance": {"id": "5", "ram": 4096, "vcpus": 2, "rxtx_factor": 400,
+                                          "disk": 160},
+               "8 GB Standard Instance": {"id": "6", "ram": 8192, "vcpus": 4, "rxtx_factor": 600,
+                                          "disk": 320},
+               "15 GB Standard Instance": {"id": "7", "ram": 15360, "vcpus": 6, "rxtx_factor": 800,
+                                           "disk": 620},
+               "30 GB Standard Instance": {"id": "8", "ram": 30720, "vcpus": 8, "rxtx_factor": 1200,
+                                           "disk": 1200}}
+
+    def extra_specs_json(self):
```

Comment:
@lekhajee the data in extra_specs_json is different for each flavor class. I would like to leave it here since it is specific info for each each flavor